gremlin-python/src/main/python/gremlin_python/driver/resultset.py

```diff
@@ -23,9 +23,8 @@
 
 class ResultSet:
 
-    def __init__(self, stream, request_id):
+    def __init__(self, stream):
         self._stream = stream
-        self._request_id = request_id
         self._done = None
         self._aggregate_to = None
         self._status_attributes = {}
```

Comment:
Nit: You should be able to safely ignore status attributes so this part could be removed.